### ISSUE TITLE
Text shadow makes other text colors appears fuzzy

### ISSUE THREAD

**Issue by xkisu** [stevenpetryk/mafs#85]
When specifying a color other than the default for the `Text` component, the edges of the text become fuzzy and appears bold

For example, here's what the default text color looks like:

```tsx
<Text
    x={yTip[0]}
    y={yTip[1]}
    attach="e"
    attachDistance={25}
>
    Y
</Text>
```
![image](https://user-images.githubusercontent.com/28452108/212241416-27a9ea12-53f7-4df9-8a7a-a3f95987f517.png)

And here's how it appears with a different color specified:
```tsx
<Text
    x={yTip[0]}
    y={yTip[1]}
    attach="e"
    attachDistance={25}
    color={Theme.green}
>
    Y
</Text>
```
![image](https://user-images.githubusercontent.com/28452108/212241483-fa9283a9-7081-4cae-acb2-1a05a19059e1.png)

This happens because when a custom color is set, the `Text` component uses that color as its stroke instead of using the background color (line https://github.com/stevenpetryk/mafs/blob/7d86d79c916606c6ea8e48dad7fce69b1e5377b4/src/display/Text.tsx#LL66). However this causes the size of the text to inflate and become fuzzy because of the simulated shadow effect from the `mafs-shadow` class not playing nice with colours other than white. 

Here's how the above example looks like with the stroke set back to `mafs-bg`:
![image](https://user-images.githubusercontent.com/28452108/212242045-81890f4c-c615-41c5-bdb7-533bb9273795.png)

**Proposed solution**: Update `Text` to still use the `mafs-bg` color as the text stroke, even when a custom color is specified.

**Comment by stevenpetryk** [comment]
Ah, good catch. Will cut a release to fix this right now.

**Comment by stevenpetryk** [comment]
<img width="675" alt="Thanks, @xkisu! This is fixed in v0.10.1." src="https://user-images.githubusercontent.com/1724000/212254628-20c9d2ef-0a73-438e-ac4f-5c61828cc495.png">

**Comment by xkisu** [comment]
> <img alt="Thanks, @xkisu! This is fixed in v0.10.1." width="675" src="https://user-images.githubusercontent.com/1724000/212254628-20c9d2ef-0a73-438e-ac4f-5c61828cc495.png">

Cheers!
